charts/argo-rollouts/templates/argo-rollouts-deployment.yaml

```diff
@@ -2,10 +2,19 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ .Release.Name }}
+  {{- if .Values.podAnnotations }}
+  annotations:
+  {{- range $key, $value := .Values.podAnnotations }}
+    {{ $key }}: {{ $value | quote }}
```

Comment:
I like that you used `| quote`. Is there any reason that you decided not to use it elsewhere in the PR?

Reply:
I think it's just to make the quotation a bit more explicit in this one section.